cap/modules/theme/templates/cap_theme/header.html

```diff
@@ -27,36 +27,90 @@
-        <a class="cap-home-logo" href="/">
-            <img src="{{ url_for('static', filename='img/cap_logo_lrg.svg')}}" onerror="this.src={{ url_for('static', filename='img/cap_logo_sm.png')}}; this.src=null;" alt="{{_(config.THEME_SITENAME)}}" height="65px" class="visible-lg-inline-block"/>
-            <img src="{{ url_for('static', filename='img/cap_logo_sm.svg')}}" height="65px;" onerror="this.src={{ url_for('static', filename='img/cap_logo_sm.png')}}; this.src=null;" alt="{{_(config.THEME_SITENAME)}}" class="hidden-lg"/>
+
+  {# logo #}
+  <div class="col-sm-2">
+    <a class="cap-home-logo" href="/">
+      <img src="{{ url_for('static', filename='img/cap_logo_lrg.svg')}}" onerror="this.src={{ url_for('static', filename='img/cap_logo_sm.png')}}; this.src=null;" alt="{{_(config.THEME_SITENAME)}}" height="65px" class="visible-lg-inline-block"/>
+      <img src="{{ url_for('static', filename='img/cap_logo_sm.svg')}}" height="65px;" onerror="this.src={{ url_for('static', filename='img/cap_logo_sm.png')}}; this.src=null;" alt="{{_(config.THEME_SITENAME)}}" class="hidden-lg"/>
+    </a>
+    <img src="{{ url_for('static', filename='img/demo.png') }}" alt="" style="position:absolute;top:0;left:0;width:200px;">
+  </div>
+
+  {%- if not current_user.is_authenticated %}
+
+    {# login button #}
+    <div class="col-sm-6"></div>
+    <div class="col-sm-4"> 
+      <div class="text-right">
+        <a href=" {{url_for('invenio_oauthclient.login', remote_app='cern', next=url_for('cap_theme.index'))}}" class="login-button btn btn-primary btn-lg" >
+          <i class="fa fa-sign-in"></i>
+          <span class="visible-lg-inline">{{_('Log in with your CERN account')}}</span>
+          <span class="visible-md-inline visible-sm-inline">{{_('CERN Login')}}</span>
         </a>
-        <img src="{{ url_for('static', filename='img/demo.png') }}" alt="" style="position:absolute;top:0;left:0;width:200px;">
-        {% endblock %}
+      </div>
     </div>
+  {% endif %}
 
-    {%- block navbar_search %}
-    <div class="col-sm-7">
-        {%- include "cap_theme/searchbar.html" %}
+  {%- if current_user.is_authenticated %}
+
+    {# searchbar #}
+    <div class="col-sm-6">
+      {%- include "cap_theme/searchbar.html" %}
     </div>
-    {%- endblock navbar_search %}
-
-    {%- block navbar_right %}
-    <div class="col-sm-3">
-        <div class="row">
-            {%- block navbar_new_deposit %}
-            <div class="col-xs-4">
-                {%- include "cap_theme/new_deposit_btn.html" %}
-            </div>
-            {%- endblock navbar_new_deposit %}
-            <div class="col-xs-8">
-                {%- include "cap_theme/header_login.html" %}
-            </div>
+
+    <div class="col-sm-4">
+
+      {# "Create An Analysis" button #}
+      <div class="col-xs-4">
+        {%- include "cap_theme/new_deposit_btn.html" %}
+      </div>
+
+      {# profile dropdown and logout button #}
+      <div class="col-xs-8 text-right">
```

Comment:
Yes, I removed header_login. It got too unreadable with all these if logged in etc. so I rearranged the whole thing, now we have two seperate blocks for two lypes of headers, one before and one after login.